extensions/amp-playbuzz/0.1/amp-playbuzz.js

```diff
@@ -181,8 +184,9 @@ class AmpPlaybuzz extends AMP.BaseElement {
       this.iframeLoaded_ = true;
       this.attemptChangeHeight(this.itemHeight_).catch(() => {/* die */ });
 
-      const unlisten = this.getViewport().onChanged(
+      const unlisten = events.listen(this.win,'scroll',
```

Comment:
from our tests some of the formats responded pretty slow and the reason was that onChanged fires only when the user has stoped the scroll. 
this might be useful in some cases but we need to expose some items while scrolling - even if the user doesn't stop scrolling
i'm looking at 'viewport.onScroll()' and it seems to work good
should we give it a try ?